openmaptiles/styleutils.py

```diff
@@ -0,0 +1,86 @@
+import json
+from pathlib import Path
+from openmaptiles.tileset import Tileset
+
+
+def fp_to_dict(fp: str) -> dict:
+    with open(fp, 'r') as f:
+        d = json.load(f)
+    return d
+
+
+def get_ts_lyr_style_json_fp(yaml_fp: str) -> Path:
+    yaml_path = yaml_fp.filename
+    json_name = Path(yaml_path.name).with_suffix('.json')
+    ts_lyr_dir = yaml_path.parent
+
+    return ts_lyr_dir.joinpath(json_name)
+
+
+def add_order(lyrs: list) -> list:
+    for i in range(0, len(lyrs)):
+        lyr = lyrs[i]
+        lyr['order'] = i
+    return lyrs
+
+
+def get_order(layer: dict) -> int:
+    return int(layer.get('order'))
+
+
+def split(tileset_fp: Path, style_fp: Path):
+    tileset = Tileset.parse(tileset_fp)
+    tileset_lyrs = tileset.layers
+
+    # Add order to each style layer (e.g. backgroud - 0, labels - 196)
+    style = fp_to_dict(style_fp)
+    style_lyrs_w_order = add_order(style.get('layers'))
```

Comment:
same - add_order should not get a None value if style has no layers.

Reply:
I added a check whether `order` is None and an error message.